docs/use/vscode.md

```diff
@@ -102,6 +102,12 @@ A notification will appear asking you to sign in using your web browser. Clickin
 
 > **Linux users:** You may be prompted to enter a user code if you are using an older version of Live Share (v0.3.295 or below). Update to the latest version of the extension or click the "Having trouble?" link after sign in to see the code. See [below for details](#sign-in-using-a-user-code).
 
+#
+
+> **Advanced tip:** The `liveshare.account` and `liveshare.accountProvider` settings enable you to select which account should be used for automatic sign-in in case you have cached credentials for multiple accounts available. 
+
+> For example, imagine you work on 2 projects with which you want to sign in with different identities. You could set the `liveshare.account` setting to different email addresses in each project directory to ensure they each automatically sign-in with the correct account. The `liveshare.accountProvider` setting could be set to either `"microsoft"` or `"github"` in case you use the same email address with multiple providers.
```

Comment:
> setting to different email addresses in each project directory

Clarify that this refers to VS Code _workspace settings_

Reply:
Done